src/core/stream/stream_events.ts

```diff
@@ -232,6 +249,13 @@ function endOfStream() : IEndOfStreamEvent {
   };
 }
 
+function resumeStream() : IResumeStreamEvent {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Update or refactor this function so that its implementation doesn't duplicate the one on line 245. ([typescript:S4144](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4144))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=229&issues=AWOmxwPqp7N0pg-JzM6w&open=AWOmxwPqp7N0pg-JzM6w)

[ISSUE_KEY]: AWOmxwPqp7N0pg-JzM6w